tensor2tensor/data_generators/text_encoder.py

```diff
@@ -121,15 +121,25 @@ def vocab_size(self):
 class TokenTextEncoder(TextEncoder):
   """Encoder based on a user-supplied vocabulary."""
 
-  def __init__(self, vocab_filename, reverse=False, num_reserved_ids=2):
+  def __init__(self, vocab_filename, reverse=False, num_reserved_ids=2, unk=None):
```

Comment:
That's indeed because of the `_DATA_FILES` constant that I implemented my own `_build_vocab`. I  definitely find more relevant to use a parameter instead. I'll do it.

Thanks for the feedbacks!